### ISSUE TITLE
[Question] Chromium telemetry?

### ISSUE THREAD

**Issue by GrabbenD** [Alex313031/thorium#144]
I currently use Brave since I value performance and privacy. From my understanding Chromium project has embedded telemetry from Google and projects like Brave and [Ungoogled-chromium](https://github.com/ungoogled-software/ungoogled-chromium) removes those calls in their releases.

Is the default telemetry stripped in this fork of Chromium?

**Comment by xplshn** [comment]
I was wondering the same, does Thorium make calls to Google, even if you haven't logged on? If so, any way to disable that default behavior?

**Comment by jstdk** [comment]
I read that there is always a call to the Chrome Webstore on launch to see if there are updates. Not sure what is in that call and/or if it is disabled in Thorium.

**Comment by xplshn** [comment]
Yeah, also te check if login with oath using Google, so it syncs your browser. The suggestions on the search box, and the privacy sandbox thing, those are just a few.

**Comment by guria9** [comment]
valid question to ask. @Alex313031 what kind of data is still being transmitted ? i know google pushes quickly a lot stuff which may fly under the radar

**Comment by kyundothost** [comment]
brave did not remove google telemetry https://spyware.neocities.org/articles/brave

**Comment by Alleop5** [comment]
There should be 0 requests made on startup. And 0 requests ever to Thorium/Google's servers. Except for when updating.

The Google Hangout extension should really be gone as it does have some kind of logging built-in:
// A port for continuously reporting relevant CPU usage information to the page.
This comment in the extension would make me think that it does have some logging through webrtc.

**Comment by ChaosNicro** [comment]
I recently switched from UGC to thorium because I neeeded a more recent chromium version. I understand that this project doesn't aim to be ungoogled, but it would be nice to have enough configuration flexibility to provide the option.
The Webstore-Manager from UGC should work in Thorium as is, but the default Auto-Update would have to be disabled.
If there is a reliance on auto-checks, such as sync or version updates, maybe having the option to switch to a manual trigger would work.

Thus taking the services one at a time. Requesting a flag-switch to turn this into UGC is unreasonable. Maybe eventually, there can be a pref-file shared to set the relevant options. But leaving those customizations to the user should not be asking to much.

**Comment by GrabbenD** [comment]
@gz83 @Alex313031 Any reason for closing this?

**Comment by gz83** [comment]
This is a batch closing operation (can also be regarded as a ping), I will reopen the issues that still have new responses

**Comment by Alex313031** [comment]
@gz83  @GrabbenD There was a guy that posted a list of URLs that Thorium still pings to. Some of these are required for Sync and the Web Store, but there may still be some sort of telemetry also going on, so I could search in the source code for those URLs and look into disabling them.

However, while I hate telemetry and try to add patches to increase security, Thorium does not attempt to be a completely "De-Googled" browser.

**Comment by gz83** [comment]
I do not think Thorium will achieve an "ungoogled" state for the time being, though I can make an effort to minimize related external connections; please let me know if you have more specific requirements.
